modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,435 @@
+--
++
+In the Redpanda Iceberg integration, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. The Iceberg integration supports two <<set-up-catalog-integration,catalog integration>> types. You can configure Redpanda to catalog files stored in the same object storage bucket or container where the Iceberg data files are located, or you can configure Redpanda to use an https://iceberg.apache.org/concepts/catalog/#decoupling-using-the-rest-catalog[Iceberg REST catalog^] endpoint to update an externally-managed catalog when there are changes to the Iceberg data and metadata.
+
+image::shared:iceberg-integration.png[]
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage, in addition to the log segments uploaded via Tiered Storage. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through many compatible data lakehouse, data engineering, and business intelligence https://iceberg.apache.org/vendors/[tools^]. 
+
+== Enable Iceberg integration
+
+To create an Iceberg table for a Redpanda topic, you must set the cluster configuration property `iceberg_enabled` to `true`, and also configure the topic property `redpanda.iceberg.mode`. You can choose to provide a schema if you need the Iceberg table to be structured with defined columns.
+
+. Set the `iceberg_enabled` configuration option on your cluster to `true`. 
++
+[,bash]
+----
+rpk cluster config set iceberg_enabled true 
+----
++
+[,bash,role=no-copy]
+----
+Successfully updated configuration. New configuration version is 2.
+----
+
+. Create a new topic.
++
+[,bash,]
+----
+rpk topic create <new-topic-name> --partitions 1 --replicas 1
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Enable the integration for the topic by configuring `redpanda.iceberg.mode`. You can choose one of the following modes:
++
+--
+* `key_value`: Creates an Iceberg table using a simple schema, consisting two columns, one for the record metadata including the key, and another binary column for the record's value.
+* `value_schema_id_prefix`: Creates an Iceberg table whose structure matches the Redpanda schema for this topic, with columns corresponding to each field. Requires that you register a schema in the Schema Registry (see next step), and that producers write to the topic using the Schema Registry wire format. Redpanda parses the schema used by the record based on the schema ID encoded in the payload header, and stores the topic values in the corresponding table columns. 
```

Comment:
The 'Requires that....` line gives a sentence fragment. Maybe you could reword to say something like.

You must register a ......and producers must write to the topic using Schema Registry wire format.